[2023-04-23 07:55:22,227] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.metadata manual__2023-04-23T07:55:09.906716+00:00 [queued]>
[2023-04-23 07:55:22,244] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.metadata manual__2023-04-23T07:55:09.906716+00:00 [queued]>
[2023-04-23 07:55:22,245] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-04-23 07:55:22,246] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-04-23 07:55:22,247] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-04-23 07:55:22,270] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): metadata> on 2023-04-23 07:55:09.906716+00:00
[2023-04-23 07:55:22,282] {standard_task_runner.py:52} INFO - Started process 591 to run task
[2023-04-23 07:55:22,291] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'multi_attack_dag', 'metadata', 'manual__2023-04-23T07:55:09.906716+00:00', '--job-id', '5934', '--raw', '--subdir', 'DAGS_FOLDER/attack_dag.py', '--cfg-path', '/tmp/tmp8qe6hvv9', '--error-file', '/tmp/tmp2pmb8585']
[2023-04-23 07:55:22,293] {standard_task_runner.py:80} INFO - Job 5934: Subtask metadata
[2023-04-23 07:55:22,398] {task_command.py:369} INFO - Running <TaskInstance: multi_attack_dag.metadata manual__2023-04-23T07:55:09.906716+00:00 [running]> on host a3dd7d962104
[2023-04-23 07:55:22,568] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=multi_attack_dag
AIRFLOW_CTX_TASK_ID=metadata
AIRFLOW_CTX_EXECUTION_DATE=2023-04-23T07:55:09.906716+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-23T07:55:09.906716+00:00
[2023-04-23 07:55:22,572] {transport.py:157} INFO - Attempting refresh to obtain initial access_token
[2023-04-23 07:55:22,579] {client.py:777} INFO - Refreshing access_token
[2023-04-23 07:55:23,414] {python.py:173} INFO - Done. Returned value was: None
[2023-04-23 07:55:23,429] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=multi_attack_dag, task_id=metadata, execution_date=20230423T075509, start_date=20230423T075522, end_date=20230423T075523
[2023-04-23 07:55:23,470] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-23 07:55:23,527] {local_task_job.py:273} INFO - 3 downstream tasks scheduled from follow-on schedule check
